css/style.css

```diff
@@ -1,19 +1,189 @@
+:root{
+    --grey-dark: #363434;
+    --grey: #adadad;
+    --white: #fff;
+    --orange: #f05a28;
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
+}
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    margin: 0;
+    padding: 0;
+    border: 0;
+    box-sizing: border-box;
+    font-weight: 400;
+}
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem 3rem;
+    border-bottom: 0.1rem solid hsl(0, 0%, 75%);
+}
+main {
+    display: flex;
+    flex-direction: column;
+    margin-top: -60px;
+ }
+
+ .logo{
+    display: block;
+    min-width: 1.5rem;
+    min-height: 1.5
+    rem;
+ }
+
+
+.list {
+    display: flex;
+    justify-content: center;
+    padding: none;
+    align-items: center;
+    flex-wrap: wrap;
+    list-style: none;
+
+ }
+
+.items{
+    display: flex;
+    padding: 0 1.5rem;
+} 
+
+.links{
+color: var(--grey);
+text-decoration: none;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+.links:hover{
+    color: var(--orange);
+}
+
+.items:first-child > a{
+color: var(--grey-darkdark);
+}
+.items:first-child > a:hover{
+    color: var(--orange);
+}
+.p1 {
+    background-image: url(/img/first-background.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    padding: 12rem 0;
+    color: var(--white);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: none;
+}
+.p1 h2 {
+    font-weight: 200;
+    font-size: 35px;
+    text-align: center;
+}
+
+.p1 h5 {
+    margin-top: -1rem;
+    margin-bottom: 2rem;
+    font-weight: 50;
+    font-size: 20px;
+}
+.p1 button {
+    background-color: var(--orange);
+    border: none;
+    padding: .8rem 1rem;
+    border-radius: 5px;
+}
+.p1 button:hover {
+    background-color: var(--grey-dark);
+}
+.p1 button a {
+    text-decoration: none;
+    color: var(--white);
+}
+section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 4rem;
```

Comment:
This is why you also needed a negative -60px margin on `main`, above.
Simpler to not style all sections, use a class selector instead and just target the sections you need to add this margin to, rather than all sections.